src/casl/casl-ability.factory.ts

```diff
@@ -803,11 +804,13 @@ export class CaslAbilityFactory {
       can(AuthOp.JobsRead, JobClass);
       can(AuthOp.JobsCreate, JobClass);
       can(AuthOp.JobsUpdate, JobClass);
+      can(AuthOp.JobsDelete, JobClass);
```

Comment:
This should not be here.
Only if you are part of the DeleteGroups, you are allowed to delete

Reply:
I would also   expect admins to be  able to delete Jobs ( while the ability to delete datasets is  on a  different level)